test/test_ops.py

```diff
@@ -704,6 +710,18 @@ def check_tensor_floating_is_differentiable(t):
             for arg in sample.kwargs.values():
                 check_tensor_floating_is_differentiable(arg)
 
+    # Reference testing for operations in complex32 against complex64.
+    # NOTE: We test against complex64 as NumPy doesn't have a complex32 equivalent dtype.
+    @ops(op_db, allowed_dtypes=(torch.complex32,))
+    def test_complex_half_reference_testing(self, device, dtype, op):
+        if not op.supports_dtype(torch.complex32, device):
+            unittest.skip("Does not support complex32")
```

Comment:
Couldn't filter in `@ops` decorator as helper to decide if a dtype is supported requires device

Reply:
I'm confused about this. Isn't setting `allowed_dtypes` in the decorator sufficient? I thought the decorator accounted for the device?

Reply:
You are right. That is sufficient! Thank you